src/views/Map/Layouts/Layout.tsx

```
@@ -49,12 +50,7 @@ export function Layout({
             {layerFeaturesPanel}
           </>
         }
-        topColumn={
-          <>
-            {breadcrumbs}
-            {toolbar}
-          </>
-        }
+        topColumn={<>{breadcrumbs}</>}
```

Biome (1.9.4)

[error] Avoid using unnecessary Fragment.

A fragment is redundant if it contains only one child, or if it is the child of a html element, and is not a keyed fragment.
Unsafe fix: Remove the Fragment


(lint/complexity/noUselessFragments)